tests/h/views/api_groups_test.py

```diff
@@ -100,12 +100,74 @@ def open_groups(self, factories):
         return [factories.OpenGroup(), factories.OpenGroup()]
 
     @pytest.fixture
-    def anonymous_request(self, pyramid_request):
-        pyramid_request.user = None
+    def authenticated_request(self, pyramid_request, factories):
+        pyramid_request.user = factories.User()
         return pyramid_request
 
+
+@pytest.mark.usefixtures('CreateGroupAPISchema',
+                         'group_service',
+                         'GroupContext',
+                         'GroupJSONPresenter')
+class TestCreateGroup(object):
+
+    def test_it_inits_group_create_schema(self, pyramid_request, CreateGroupAPISchema):  # noqa: N803
```

Comment:
Are these `# noqa: N803`'s necessary? `N803` is [disabled globally in our flake8 config](https://github.com/hypothesis/h/blob/cea953949d1bd68a9baeed292e1b4fa0e602f1f7/tests/setup.cfg#L13) (so is `N802`)